dbdownload/dbdownload.py

```diff
@@ -176,10 +183,8 @@ def _monitor(self):
                 if changed and self.executable:
                     self._launch(self.executable)
 
-                # Done processing delta, sleep and check again.
-                tree = {}
-                self._logger.debug('sleeping for %d seconds' % self.sleep)
-                time.sleep(self.sleep)
+                # Done processing delta
+                return
```

Comment:
Since the "while True" is in start() now, there's no reason to have a "while True" loop in _monitor.  Remove the _monitor while loop, and I think these changes should be good to go.